CONTRIBUTING.rst

```diff
@@ -0,0 +1,88 @@
+Contributing
+============
+
+RGBDS was created in the late 90's and has received contributions of several
+developers since then. It wouldn't have been possible to get to this point
+without their work and. For that reason, it is always open to the contributions
```

Comment:
“...without their work, and for that reason, it...”

Reply:
“...without their work and, for that reason, it...” :P